index.bs

```diff
@@ -5401,6 +5407,30 @@ authenticator=] and [=client device=] SHOULD work together to ensure that the ex
 only to the operating system user that created that [=platform credential=].
 
 
+## Privacy of [PII] stored in authenticators ## {#sctn-pii-privacy}
+
+[=Authenticators=] MAY provide additional information to [=clients=] outside what's mandated by this specification &mdash; for
```

Comment:
" &mdash;" -> ","

Reply:
Ok, I also changed "for example," to "e.g.," here to lessen the risk of the "for example" being read as a modifier for "Authenticators MAY provide additional information".